### PR TITLE
Work on the UI: display the jobs retrieved as a table

### PR DESCRIPTION
**Pull request by andreeanica16**
1. All the information fetched from the JobsServlet will be displayed as a table when pressing "Show Info" button.

### CODE REVIEW

**Comment by andreeanica16** [comment]
@juliocc  @iht  @tblanshard

File: src/main/webapp/script.js [hunk 1]
```diff
@@ -43,5 +43,92 @@ function updateProjectDatabase() {
...
   fetch('/jobs?projectID=' + projectID)
   .then(response => response.json())
-  .then(jobs => console.log(jobs))
+  .then(jobs => {
+      console.log(jobs);
+      const table = document.getElementById('jobs');
+      for (let i = 0; i < jobs.length; i++) {
+          addJobToTable(jobs[i], table);
+      }
+      table.hidden = false;
```
Comment (tblanshard) [inline src/main/webapp/script.js:52]: I might have missed something, but is the table hidden at any point?

Comment (andreeanica16) [reply src/main/webapp/script.js:52]: Yes, the table is hidden before pressing ShowInfo button. Hope this answer your question :D

Comment (tblanshard) [reply src/main/webapp/script.js:52]: Okay thanks - yes it does :)

**Pull request merged**